flake.nix

```diff
@@ -5,10 +5,6 @@
 
   outputs = { self }:
     let
-      jobs = import ./pkgs/top-level/release.nix {
-        nixpkgs = self;
-      };
```

Comment:
You're now evaluating the release.nix file twice in the flake. Something like

```nix
jobs = forAllSystems (system: import ./pkgs/top-level/release.nix {
  nixpkgs = self;
  inherit system;
});
```

And then

```nix
checks = forAllSystems (system: {
  tarball = jobs.${system}.tarball
# ...
```

avoids that :)

Reply:
Woops, well that is fixed. I did the same for the other occurrence.